### ISSUE TITLE
custom dialect class not found

### ISSUE THREAD

**Issue by jgribonvald** [liquibase/liquibase-hibernate#67]
It would be great to include project classpath to be able to use custom dialect. The workaround is to include as dependency the project jar generated, but it's a bit odd to do like that and after each code change we should generate a package.

**Comment by tomhughesnice** [comment]
Has any progress been made on this issue as im having exactly the same problem.
Using Liquibase maven plugin version 3.4.1 with liquibase hibernate 4 version 3.5

I am using a Postgres database, connecting with hibernate. I have had to create my own Dialect class to handle Postgres's 'JSONB' type. 
I have updated the particular line in the maven plugin 
                    ```<referenceUrl>hibernate:spring:com.fieldmargin.fm.core.domain?dialect=com.fieldmargin.fm.core.config.dialect.JsonPostgreSQLDialect</referenceUrl>```
But I always get a class not found exception when trying to load the dialect.

Its strange as I can see in the liquibase debug the classpath is being loaded, and the Dialect class I created is in there. So there must be a bug in the Liquibase class loader for the dialect.

The work around above gets me past this issue. I have to run a Maven install, and make the produced artifact a dependancy of the liquibase plugin. Would be good to get a proper fix though.
